### PR TITLE
Pptp 2609 liability accessability

### PR DESCRIPTION
**Pull request by lmcewanhmrc**
Liability changes fix A11y issues

### CODE REVIEW

File: app/views/components/headingLabel.scala.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@this()
+
+@(headingContent: String, forID: String = "value")
+
+<h1 class="govuk-label-wrapper">
```
Comment (liampan) [inline app/views/components/headingLabel.scala.html:21]: this is no longer used. 💀

**Review by liampan — COMMENTED** [review]
//todo 
 final content
 welsh content
 Errors

File: app/views/components/headingLabel.scala.html (hunk 1 above)
Comment (lmcewanhmrc) [reply app/views/components/headingLabel.scala.html:21]: ☠️

File: conf/messages.en [hunk 2]
```diff
@@ -442,37 +440,45 @@ notLiable.think.info = This is a new service and we want to know what you think
...
 liability.exceededThresholdWeight.line3 = If you’re registering as a group, each member must have met this threshold.
-liability.exceededThresholdWeight.beforeApril2023.question.empty.error = Select yes if you have manufactured or imported 10,000kg or more finished plastic packaging since 1 April 2022
-liability.exceededThresholdWeight.question.empty.error = Select yes if you have manufactured or imported 10,000kg or more of finished plastic packaging in the last 12 months
-
-liability.exceededThresholdWeightDate.sectionHeader = Liability details
-liability.exceededThresholdWeightDate.title = When did you meet the 10,000kg threshold?
+liability.exceededThresholdWeight.line4 = If you’re unsure, find out more about {0}.
+liability.exceededThresholdWeight.line4.link-text = the forward and backward look tests
+liability.exceededThresholdWeight.question = Have you met the threshold in the last 12 months?
+liability.exceededThresholdWeight.question.empty.error = TODO
```
[the remark this answers is not in the document]
Comment (lmcewanhmrc) [reply conf/messages.en:476]: Dunno

**Pull request merged**